streams/integration-tests/src/test/java/org/apache/kafka/streams/integration/KafkaStreamsTelemetryIntegrationTest.java

```diff
@@ -496,16 +483,20 @@ private void addGlobalStore(final StreamsBuilder builder) {
                 () -> new Processor<>() {
                     private KeyValueStore<String, String> store;
```

Comment:
Seems we don't need this as a variable?

Reply:
True, since we're not processing any records.  I've removed it here, declared in the `init` method.